README.md

```diff
@@ -176,6 +176,22 @@ In your application, at the point which the view is ready for rendering
 document.body.dispatchEvent(new Event('view-ready'))
 ```
 
+**Warning:** It is possible that your application will be ready and emit the event before the main electron process has had a chance execute the javascript in the renderer process which listens for this event.  
+
+If you are finding that the [event is not effective](https://github.com/fraserxu/electron-pdf/issues/169) and your page waits until the full timeout has occurred, then you should use `setInterval` to emit the event until it is acknowledged like so:
+
+```javascript
+  var eventEmitInterval = setInterval(function () {
+    document.dispatchEvent(new Event('ready'))
```

Comment:
I see this event is called `ready` and the other is `view-ready-ack`.  Did you intend for it to be just `ready-ack`?  Also, I think it would be worth the extra few characters for the suffix to be `acknowledged` or `heard` or something so the code is more self-documenting.